bin/setup

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env ruby
 require "fileutils"
 
-# path to your application root.
 APP_ROOT = File.expand_path("..", __dir__)
+APP_NAME = "manage-vaccinations-in-schools"
```

Comment:
Interesting, I had spotted the `puma-dev` lines but left them out since they were commented out, I'll remove `APP_NAME` for now, we can always bring it back if we decide to use it.